ledfx/effects/metro.py

```diff
@@ -0,0 +1,104 @@
+import logging
+import timeit
+
+import numpy as np
+import voluptuous as vol
+
+from ledfx.color import parse_color, validate_color
+from ledfx.effects.temporal import TemporalEffect
+
+_LOGGER = logging.getLogger(__name__)
+
+# This effect is intended to assist in obvservations of synchronisity from the
```

Comment:
All comments need to be rewritten for final implementation, they were where i started

Reply:
done